pipeline/tools/amass.yaml

```diff
@@ -1,7 +1,8 @@
 installed: false
 dependencies: [go]
-go: &gotool !get_tool_path "{go}"
-amass: &amass !get_tool_path "{amass}"
+tools: &tools !get_default "{tools-dir}"
+go: &gotool !get_tool_path "{go[tool_path]}"
+tool_path: &amass !join_path [*tools, "amass"]
 
 commands:
 - !join [*gotool, get -u github.com/OWASP/Amass/v3/...]
```

Comment:
Can't put a comment on the actual line, but `shell: true` isn't necessary for amass; can remove